src/WebJobs.Extensions.DurableTask/DurableOrchestrationStatus.cs

```diff
@@ -11,7 +11,7 @@ namespace Microsoft.Azure.WebJobs.Extensions.DurableTask
     /// </summary>
     /// <remarks>
     /// An external client can fetch the status of an orchestration instance using
-    /// <see cref="IDurableOrchestrationClient.GetStatusAsync(string, bool, bool, bool)"/>.
```

Comment:
If we follow my suggestion to reuse `showInput`, then we can revert all changes to this file.